textproc/moar/Portfile

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/walles/moar 1.8.4 v
+revision            0
+
+description         Moar is a pager. It's designed to just do the right thing without any configuration.
+
+long_description    Moar should work as a drop-in replacement for Less. \
+                    It features syntax hightlighting, ANSI color coded text support, \
+                    incremental search and automatic decompression, among others.
+
+homepage            https://github.com/walles/moar
```

Comment:
You are using the `golang` portgroup, which uses the `github` portgroup when the Go package lives in GitHub.  As a result of this, the `homepage` is automatically set to match the package provided in `go.setup`.  So it's not necessary to explicitly set `homepage` here.